README.md

```diff
@@ -151,56 +153,71 @@ a gallery from uploaded medias.
 
 In the following i provide an example:
 
-
 In your `MediaAdminController.php`:
 ```php
-    /**
-     * @return RedirectResponse
-     */
-    public function createGalleryAction(Request $request)
+namespace App\Controller;
+
+use SilasJoisten\Sonata\MultiUploadBundle\Controller\MultiUploadController;
+use Sonata\MediaBundle\Entity\MediaManager;
+use Sonata\MediaBundle\Entity\GalleryManager;
+
+class MediaAdminController extends MultiUploadController
```

Comment:
Great improvement of the example.

Consider making the controller class final.

Good to merge